### DISCUSSION TITLE
release a template for jquery-ui

### DISCUSSION THREAD

**Discussion by cbaTW** [lisonge/vite-plugin-monkey#81] [General]
I've been tried many way to use jquery-ui under Vue template because I need the draggable() function.

I found some version of jquery-ui is not compatible with jquery LTS in StackOverflow, so I downgrade to jquery-ui 1.12.1 and jquery 2.2.4.

It fix the problem "jQuery is not defined" but the draggable() still not works.

Please help me to use **`Vue 3, BootStrap 5, jQuery, jQuery-UI`** simultaneously, it would help me a lot if you release a template with `package.json, vite.config.js, App.vue, main.js`

**Comment by lisonge** [comment]
emmm, jQuery is already out of date

maybe you can use 

- <https://github.com/SortableJS/vue.draggable.next>
- <https://vueuse.org/core/useDraggable>
